src/scss/layouts/_header.scss

```
@@ -0,0 +1,78 @@
+.header {
+  @include absolute();
+  background-color: rgba($color: $color-white, $alpha: 0.1);
+  z-index: calc($zindex1 + 1);
+  width: $wf;
+  &__list {
+    @include flex($justify: flex-start, $align: center, $gap: 1rem);
+    @include respond-to(desktop-and-down) {
+      display: none;
+    }
+  }
+  &__item {
+    @include relative;
+    &:not(:first-child)::after {
```

Check lại file này nhé
```suggestion
  }
  
  &__item {
    @include relative;
    
    &:not(:first-child)::after {
```

---

mỗi  cái & đó cũng phải trống 1 hàng à a